src/DigitalAssets.js

```
@@ -185,4 +185,25 @@ export default class DigitalAssetsService extends Service {
       'POST'
     )
   }
+
+  /**
+   * Get application installer resource.
+   * @param  {Object} param
+   * @param  {HostApplicationName} param.hostAppName
+   * @param  {String} param.hostAppVersion This should be a semantic version number string, or `latest`.
+   * @param  {HostOs} param.hostOs
+   * @param  {String} param.includeCorePack Used to distinguish between corepack and non corepack downloads.
+   * Only applicable fot `studio`.Ignored for other products.
+   * @return {Promise<DownloadEmail>}
+   */
+  getApplicationInstallerResource ({ hostAppName, hostAppVersion, hostOs, includeCorePack = true }) {
```

just wondering whether `includeCorePack` being true by default makes sense - since value feels additive (it adds something extra when true) it almost feels like it should be false by default

I'm not too bothered either way, just thinking about what makes the most sense from an outside perspective

---

Yah I was not too confidant about the default value too. But if its not provided, it will be treated as `false`. Since corepack installer is the default download in the download page, i kept this `true`

---

I think this should be false by default, otherwise it's a bit misleading because an `include_core_pack` value would be in the body when a value is not passed in as parameter?

---

Mmm. `include_core_pack` will be in the body regardless of the value we passed to `includeCorePack`. Now that I think about again, I also feel `includeCorePack = true ` is not needed. we can remove the default value and it will be treated as a false by default.

---

Removed the default `true` value for `includeCorePack` param

---

Thanks Sameera